.github/workflows/precommit.yaml

```diff
@@ -35,7 +43,7 @@ jobs:
         run: |
           set -ex
           python -m pip install --upgrade pip wheel
-          python -m pip install -e .[dev]
+          python -m pip install -e .${{ matrix.install_version }}
       - name: Linting
         run: |
           pre-commit run --all-files
```

Comment:
IIRC you have to do a `pre-commit install` before you can run or does this go ahead and install `pre-commit` as well?

Reply:
pre-commit is getting installed as part of [dev](https://github.com/mosaicml/streaming/blob/main/setup.py#L47) in setup.py file and all the dependency linting will get install inside command `pre-commit run --all-files`.